core/src/main/java/cz/o2/proxima/transform/DelegatedTransform.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2017-2020 O2 Czech Republic, a.s.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.o2.proxima.transform;
+
+import com.google.common.base.Preconditions;
+import cz.o2.proxima.annotations.Internal;
+import cz.o2.proxima.repository.DataOperator;
+import cz.o2.proxima.repository.DataOperatorFactory;
+import java.io.Serializable;
+
+@Internal
+public interface DelegatedTransform extends Serializable {
+
+  /** Verify if this {@link DataOperator} specific (contextual) transform. */
+  default boolean isContextualTransform() {
```

Comment:
Hmm, `isContextualTransform` in `DelegatedTransform`, typo?

Reply:
I'm seeing it now, this is parent of ContextualTransform...

Reply:
I changed this to be more readable.

Reply:
Thx